src/Resources/app/administration/src/module/findologic-module/components/findologic-config/index.js

```diff
@@ -1,130 +1,150 @@
-import template from './findologic-config.html.twig';
+import template from './findologic-config.html.twig'
```

Comment:
Please add `;` to the end of lines also for JavaScript. Internally we also do this for our JavaScript projects, so let's keep the convention.